docs/reference-implementation/quality-assurance.md

```diff
@@ -59,39 +59,39 @@ Testing is organised across multiple levels to validate functionality, performan
 - **Tools:** [Performance Testing Tools](#test-tools).
 
 ## Test Tools
 
-The following table summarises the tools utilised in the applicable test types:
+The following table summarises the tools used in the applicable test types:
 | Test Type | Tools | 
 | --- | --- | 
 | Unit | _SonarCloud_ |
 | Functional | _Manual_, _Serenity BDD_ for structured reporting and visual coverage, Automation: Appium (for mobile automation on Android and iOS), Java (test scripting language), JUnit (test framework), IntelliJ IDEA (development and execution environment), Maven (dependency and build management), Appium Inspector (for element inspection and locator validation), Xcode (for building and exporting iOS .ipa files and Android .apk packages retrieved manually from Firebase for testing). |
 | Security | _OWASP MASVS/MASTG_ for mobile security verification <br> _OWASP Dependency-Check_ for third-party library analysis <br> _SQLCipher_ and _Android Keystore_ for data protection <br> _Burp Suite_ and _MobSF_ for dynamic testing and static analysis|
-| Performance | Android: Android Studio Profiler and App Inspection and iOS: Xcode Instruments for performance monitoring, memory usage, and CPU/network profiling |
+| Performance | Android: Android Studio Profiler and App Inspection and iOS: Xcode Instruments for performance monitoring, memory usage and CPU/network profiling |
 
 
 ## Reporting
 
-Each release is accompanied with serenity-generated reports summarising test execution results and coverage.
+Each release is accompanied by serenity-generated reports summarising test execution results and coverage.
 
 ### Release Test Reports
 
 | Release | Test Types | Report Links | Status |
 | --- | --- | --- | --- |
-| 2025.07.28 - Demo | Functional <br/> Security <br/> Performance | [Functional test results](https://mgiakkou.github.io/testing/2025_q3/) <br/> [Security test results](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/raw/refs/heads/milestone/2025.Q4/doc/security/EUDI-Wallet-NiScy_Security%20Testing%20Results-Version_2025.07.28-Demo.xlsx) <br/> [Performance test results](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/blob/milestone/2025.Q4/doc/performance/preformance_test_results.md)| ✅ Released |
+| 2025.07.28 - Demo | Functional <br/> Security <br/> Performance | [Functional test results](https://mgiakkou.github.io/testing/2025_q3/) <br/> [Security test results](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/raw/refs/heads/milestone/2025.Q4/doc/security/EUDI-Wallet-NiScy_Security%20Testing%20Results-Version_2025.07.28-Demo.xlsx) <br/> [Performance test results](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/blob/milestone/2025.Q4/doc/performance/performance_test_results.md)| ✅ Released |
```

Comment:
The suggestion is correct, but we need also to change the target where the file is also misspelled there. 

If not the correctly spelled, on our side, link stays broken

<img width="1136" height="568" alt="Image" src="https://github.com/user-attachments/assets/1cb802f3-2b06-4c46-a5c0-a058bc474463" />